LanguageFeatures/Inline-classes/syntax_A07_t01.dart

```diff
@@ -2,62 +2,60 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-/// @assertion A rule for <viewDeclaration> is added to the grammar, along with
-/// some rules for elements used in view declarations:
+/// @assertion A rule for <inlineClassDeclaration> is added to the grammar,
+/// along with some rules for elements used in inline class declarations:
 ///
-/// <viewDeclaration> ::=
-///   'view' 'class' <typeIdentifier> <typeParameters>?
-///       <viewPrimaryConstructor>?
-///       <interfaces>?
+/// <inlineClassDeclaration> ::=
+///   'inline' 'class' <typeIdentifier> <typeParameters>? <interfaces>?
 ///   '{'
-///     (<metadata> <viewMemberDeclaration>)*
+///     (<metadata> <inlineMemberDeclaration>)*
 ///   '}'
 ///
-/// <viewPrimaryConstructor> ::=
-///   '(' <type> <identifier> ')'
-///
-/// <viewMemberDeclaration> ::=
-///   <classMemberDefinition>
+/// <inlineMemberDeclaration> ::= <classMemberDefinition>
 /// ...
-/// That is, every view declares exactly one instance variable, and it is final.
-/// A primary constructor (as defined in this document) is just an abbreviated
-/// syntax whose desugaring includes a declaration of exactly one final instance
-/// variable.
-/// ```dart
-/// // Using a primary constructor.
-/// view class V1(R it) {}
-///
-/// // Same thing, using a normal constructor.
-/// view class V2 {
-///   final R it;
-///   V2(this.it);
-/// }
-/// ```
-/// There are no special rules for static members in views. They can be declared
-/// and called or torn off as usual, e.g., View.myStaticMethod(42).
+/// There are no special rules for static members in inline classes. They can be
+/// declared and called or torn off as usual, e.g., Inline.myStaticMethod(42)
 ///
-/// @description Checks that it is a compile-time error if a view declaration
-/// has <viewPrimaryConstructor> and declares a member with the name equal to
-/// the representation name
+/// @description Checks that inline classes may have static members
```

Comment:
Perhaps add something like 'Note that static members are declared before instance members (and before the unique instance variable). This is not likely to be the style which is recommended, but it should be used in at least one test.'